integration/config.sh

```diff
@@ -41,7 +41,7 @@ has() {
     local host=$2
     local name=$3
     local count=${4:-1}
-    assert "curl -s http://${host}:4040/api/topology/${view}?system=show | jq -r '[.nodes[] | select(.label == \"${name}\")] | length'" "$count"
+    assert "curl -s http://${host}:4040/api/topology/${view}? | jq -r '[.nodes[] | select(.label == \"${name}\")] | length'" "$count"
 }
```

Comment:
@2opremio This ended up getting my tests to pass on this branch, but did I remove the purpose of the test?